Training_Preparation.md

```diff
@@ -0,0 +1,275 @@
+As a response, we receive the contract ID.
+
+
+#### d) Requesting a Pull Transfer
+We provide the contract ID.
+
+
+#### e) Requesting an EDR
+The EDR contains the data (authentication information and URL) with which we can subsequently access the asset. The authentication token also only has a validity duration of a few minutes. Please note that the EDR may need to be renewed by re-executing this request.
+
+
+#### f) DTR Lookup API via EDC Transfer
+We now send a lookup request to the partner's DTR, but the EDC transfer is used as a proxy for this. This means we use the base URL from the EDR and the URL paths corresponding to the DTR API. Additionally, we add the base64-encoded query. Both are forwarded by the EDC to the DTR. Accordingly, the response should be identical to what we saw in Task 1 c.  
+
+By the way, do you remember the HTTP-header named "edc-bpn", which we were looking at during task 1? Is it needed here too? 
+
+
+#### g) Querying a Specific Shell via EDC Transfer
+This step corresponds to the step from Task 1 d, but using an EDC transfer. Note that we can use the same contract as the basis for all requests. We can even use the same transfer that we requested in Task 3 d. Only the token may need to be renewed if it has expired.
+
+
+#### h) Catalog Query for the PartTypeApi Asset
+
+
+First, let's look again at the response with the shell descriptor from the previous query. It is recommended to copy and paste the content into the code editor. Then, if we search through the "submodelDescriptors" array, we find an entry that contains the following: "urn:samm:io.catenax.part_type_information:1.0.0#PartTypeInformation".
+
+
+Let's take a closer look at this section. Within this JSON object, there is also an "endpoints" array. The entries "href" and "subprotocolBody" are particularly interesting. Evaluating this information allows us to know where and how we can access the submodel described in this shell descriptor.
+
+
+We want to spare you the trouble of copying these data individually and transferring them to other request bodies.
+
+
+If we now look at the current request body, we see that we use the string "urn:samm:io.catenax.part_type_information:1.0.0#PartTypeInformation" as a filter criterion to start another catalog query.
+
+#### i) Negotiation for a Contract over the PartType Asset
+
+
+Business as usual :)
+
+
+#### j) Querying the Negotiation Status
+As a response, we receive the contract ID.
+
+
+#### k) Requesting a Pull Transfer
+We provide the contract ID.
+
+
+#### l) Requesting an EDR
+As above.
+
+
+#### m) Pull Data from PartType-API
+
+
+We now see what information the PartType-API provides regarding this material. It's also interesting to note that the PURIS backend internally handled the response. If we take a quick look at the logs of our PURIS backend containers, we can see that the PartTypeInformationController has processed a request.
+
+
+### Task 4: Creating a New Sample Asset on Our EDC
+
+
```

Comment:
short introduction needed with some metdata on material and role (we're supplier)